### PR TITLE
feat(TMC-22423/TabBar): tab bar with labels

### PR DESCRIPTION
**Pull request by VolodymyrKovalM**
**What is the problem this PR is trying to solve?**
https://jira.talendforge.org/browse/TMC-22423

**What is the chosen solution to this problem?**

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [x] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

### CODE REVIEW

File: packages/components/src/Actions/ActionDropdown/ActionDropdown.component.js [hunk 1]
```diff
@@ -64,17 +66,28 @@ function renderMutableMenuItem(item, index, getComponent) {
...
+	const title = item.title || item.label;
+
 	return (
 		<Renderers.MenuItem
 			{...item}
 			key={index}
 			eventKey={item}
 			onClick={wrapOnClick(item)}
-			title={item.title || item.label}
+			title={item.badge && item.badge.label ? `${item.badge.label} ${title}` : title}
```
Comment (smouillour) [inline packages/components/src/Actions/ActionDropdown/ActionDropdown.component.js:78]: could you use lodash/get for it

File: packages/components/src/TabBar/TabBar.component.js [hunk 2]
```diff
@@ -127,6 +129,12 @@ function TabBar(props) {
 			<ActionDropdown
 				id={id}
 				className={classnames(theme['tc-tab-bar-dropdown'], 'tc-tab-bar-dropdown')}
+				badge={selectedItem.badge}
+				tooltipLabel={
+					selectedItem.badge && selectedItem.badge.label
```
Comment (smouillour) [inline packages/components/src/TabBar/TabBar.component.js:134]: could you use lodash/get for it?

**Review by smouillour — COMMENTED** [review]
small change required. the rest looks good to me

**Review by yyanwang — APPROVED** [review]

File: packages/components/src/Actions/ActionDropdown/ActionDropdown.component.js [hunk 3]
```diff
@@ -68,14 +69,15 @@ function renderMutableMenuItem(item, index, getComponent) {
 	}
 
 	const title = item.title || item.label;
+	const badgeLabel = get(item, 'badge.label', '');
```
Comment (smouillour) [inline packages/components/src/Actions/ActionDropdown/ActionDropdown.component.js:72]: why create a new var for that? you can set as default value: undefined or null

File: packages/components/src/TabBar/TabBar.component.js [hunk 4]
```diff
@@ -125,12 +126,13 @@ function TabBar(props) {
...
 	if (responsive && showDropdown) {
 		const selectedItem = items.find(item => item.key === selectedKey) || items[0];
+		const badgeLabel = get(selectedItem, 'badge.label', '');
 		tabMenu = (
 			<ActionDropdown
 				id={id}
 				className={classnames(theme['tc-tab-bar-dropdown'], 'tc-tab-bar-dropdown')}
 				badge={selectedItem.badge}
-				tooltipLabel={selectedItem.badge && selectedItem.badge.label ? `${selectedItem.badge.label} ${selectedItem.label}` : selectedItem.label}
+				tooltipLabel={badgeLabel ? `${badgeLabel} ${selectedItem.label}` : selectedItem.label}
```
Comment (smouillour) [inline packages/components/src/TabBar/TabBar.component.js:135]: why create a new var for that? you can set as default value: undefined or null

**Review by smouillour — APPROVED** [review]

**Pull request merged**